Jenkinsfile.deploy

```
@@ -286,11 +298,14 @@ void setDeployPropertyIfNeeded(String key, def value){
     }
 }
 
-MavenCommand getMavenCommand(String directory = 'kogito-examples'){
+MavenCommand getMavenCommand(String directory = ''){
+    if(!directory) {
+        directory = getRepoName()
+    }
```

I guess for clarity in the code, we should put the `directory` parameter as mandatory ...
```suggestion
MavenCommand getMavenCommand(String directory){
```